### ISSUE TITLE
Configure DBUnit datatypeFactory

### ISSUE THREAD

**Issue by rmpestano** [rmpestano/dbunit-rules#34]
We need to set datatype config to avoid potential issues. Currently we're receiving this warning:

```
WARNING: Potential problem found: The configured data type factory 'class org.dbunit.dataset.datatype.DefaultDataTypeFactory' might cause problems with the current database 'HSQL Database Engine'
```
